src/gulcalc/gulcalc.cpp

```diff
@@ -133,100 +227,204 @@ void splittiv(std::vector<gulItemIDLoss> &gulitems, OASIS_FLOAT tiv)
 	}
 }
 
-// because this is sparsely populated - if we keep a list of the populated fields in another vector then we do not need to iterate all the rows to clear it.
-// std::vector<std::vector<std::vector<gulItemIDLoss>>> mode1_;
-void gulcalc::clearmode1_data()
-{
+// As this is sparsely populated, if we keep a list of the populated fields in
+// another vector then we do not need to iterate all the rows to clear it
+void gulcalc::clearmode1_data() {
+
 	auto iter = mode1UsedCoverageIDs_.begin();
 	while (iter != mode1UsedCoverageIDs_.end()) {
-		mode1_[*iter].clear();
-		//auto iter2 = mode1_[*iter].begin()	;
-		//while (iter2 != mode1_[*iter].end()) {
-		//	iter2->clear();
-		//	iter2++;
-		//}
+		mode1_stats_[*iter].clear();
 		iter++;
 	}
 	mode1UsedCoverageIDs_.clear();
 }
-void gulcalc::clearfullCorr_data()
-{
-	auto iter = fullCorr_.begin();
-	while (iter != fullCorr_.end()) {
-		auto iter2 = iter->begin();
-		while (iter2 != iter->end()) {
-			iter2->clear();
+
+void gulcalc::WriteMode1Output(const int event_id, const OASIS_FLOAT tiv,
```

Comment:
Use the same casing rules for all the code in one class

Reply:
Understood; will change this.